public/js/util/storeAccessor.js

```diff
@@ -0,0 +1,9 @@
+let savedStore;
+
+export function setStore(store) {
```

Comment:
We only use it to get config out of the store in non-react utils. If we can do that in an alternative way... then yes!